test/unit/uk/gov/hmrc/apinotificationqueue/controller/QueueControllerSpec.scala

```diff
@@ -18,22 +18,33 @@ package uk.gov.hmrc.apinotificationqueue.controller
 
 import play.api.http.Status
 import play.api.test.FakeRequest
-import play.api.http.Status
-import play.api.test.FakeRequest
-import uk.gov.hmrc.play.test.UnitSpec
-import uk.gov.hmrc.play.test.WithFakeApplication
 import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
 
-class MicroserviceHelloWorldControllerSpec extends UnitSpec with WithFakeApplication{
+class QueueControllerSpec extends UnitSpec with WithFakeApplication {
```

Comment:
Do we need also a "get message by clientId" case?

Reply:
I don't think so as we don't have that functionality and won't need it with the change to have a GET All then a DELETE by ID